icekit_events/page_types/eventlistingfordate/migrations/0005_auto_20161201_1518.py

```diff
@@ -0,0 +1,20 @@
+# -*- coding: utf-8 -*-
```

Comment:
Careful with committing unrelated migrations. These should have been part of an earlier commit (prolly my bad), but if they form part of branches then they're likely to conflict with the develop branch when merged, in a way that can't be easily resolved across all installs. Best solution here is to harangue me about it :)